irr/src/WeightBuffer.cpp

```diff
@@ -0,0 +1,121 @@
+#include "WeightBuffer.h"
```

Comment:
Missing license (Irrlicht?) and copyright notice - after all, this is your code.